addons/mail/models/mail_activity_mixin.py

```diff
@@ -71,7 +71,7 @@ def _default_activity_type(self):
         'mail.activity.type', 'Next Activity Type',
         related='activity_ids.activity_type_id', readonly=False,
         search='_search_activity_type_id',
-        groups="base.group_user")
+        groups="base.group_user", index="not null")
```

Comment:
This field is not stored. Why adding an index on it ?

Reply:
A mistake, I probably put one the wrong next_activity_type when reviewing index stats.